### PR TITLE
Mdpx switch apps commands

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Switching calls to new client calls

Fixes https://github.com/meroxa/mdpx/issues/583

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [x]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||

### CODE REVIEW

File: utils/display/display.go [hunk 1]
```diff
@@ -58,11 +64,13 @@ func PrintList(input interface{}, details Details) string {
 	for _, o := range objs {
 		bytes, err := json.Marshal(o)
 		if err != nil {
+			fmt.Printf("err1: %v\n", err)
```
Comment (nassor) [inline utils/display/display.go:67]: what `err1` & `err2` stand for?

File: cmd/meroxa/root/apps/apps.go [hunk 2]
```diff
@@ -30,6 +31,34 @@ import (
...
+
+const (
+	ApplicationStateInitialized ApplicationState = "initialized"
+	ApplicationStateDeploying   ApplicationState = "deploying"
+	ApplicationStatePending     ApplicationState = "pending"
+	ApplicationStateRunning     ApplicationState = "running"
+	ApplicationStateDegraded    ApplicationState = "degraded"
+	ApplicationStateFailed      ApplicationState = "failed"
+
+	// collectionName = "apps".
```
Comment (raulb) [inline cmd/meroxa/root/apps/apps.go:44]: I assume this is not needed? Maybe we could remove in that case.

File: cmd/meroxa/root/apps/apps.go [hunk 3]
```diff
@@ -30,6 +31,34 @@ import (
...
+//var displayDetails = display.Details{"Name": "name", "State": "state", .
+//"SpecVersion": "specVersion", "Created": "created", "Updated": "updated"}.
+
+// Application represents the Meroxa Application type within the Meroxa API.
+type Application struct {
+	ID          string           `json:"id"`
+	Name        string           `json:"name"`
+	State       ApplicationState `json:"state"`
+	Spec        string           `json:"spec"`
+	SpecVersion string           `json:"specVersion"`
```
Comment (raulb) [inline cmd/meroxa/root/apps/apps.go:56]: I assume this is part of what we're already discussing in #mdpx, but I think we should continue using snakecase unless there's a very good reason to do so.

```suggestion
	SpecVersion string           `json:"spec_version"`
```

File: cmd/meroxa/root/apps/apps.go [hunk 4]
```diff
@@ -58,11 +87,9 @@ func (*Apps) SubCommands() []*cobra.Command {
 		builder.BuildCobraCommand(&Describe{}),
 		builder.BuildCobraCommand(&Init{}),
 		builder.BuildCobraCommand(&List{}),
-		builder.BuildCobraCommand(&Logs{}),
```
Comment (raulb) [inline cmd/meroxa/root/apps/apps.go:61]: I'd imagine the idea is to remove this for now until we have a better way to pull logs from mdpx?

**Review by raulb — APPROVED** [review]
@anna-cross looks good. I also wonder the same as @nassor here https://github.com/meroxa/cli/pull/833/files#r1368925587.

File: cmd/meroxa/root/apps/apps.go (hunk 2 above)
Comment (anna-cross) [reply cmd/meroxa/root/apps/apps.go:44]: This is not needed in this pr, i'll remove it for now!

File: utils/display/display.go (hunk 1 above)
Comment (anna-cross) [reply utils/display/display.go:67]: Good call out, I honestly dont remember why the errors are numbered....I'll remove the numbers

File: cmd/meroxa/root/apps/apps.go (hunk 4 above)
Comment (anna-cross) [reply cmd/meroxa/root/apps/apps.go:61]: Yup!!

**Review by raulb — APPROVED** [review]

**Pull request merged**